lgc/patch/PatchLoopMetadata.cpp

```diff
@@ -216,7 +218,7 @@ bool PatchLoopMetadata::runOnLoop(Loop *loop, LPPassManager &loopPassMgr) {
     }
   }
 
-  if (m_disableLicm) {
+  if ((m_disableLicmThreshold > 0) && (loop->getNumBlocks() >= m_disableLicmThreshold)) {
```

Comment:
nit: don't wrap and operands with parens

Reply:
Fixed